src/utils.c

```diff
@@ -0,0 +1,26 @@
+#include <string.h>
+#include "../include/utils.h"
+
+Command parseStringToCommand(char commandAsString[]) {
+    char str[strlen(commandAsString)];
+    strcpy(str, commandAsString);
+    char *delimiter = " ";
+    char *token;
+    Command command = {"", "", ""};
+    token = strtok(str, delimiter);
+    int i = 0;
+    while (token) {
+        if (i == 0) {
+            strcpy(command.order, token);
+            i++;
+        } else if (i == 1) {
+            strcpy(command.key, token);
+            i++;
+        } else {
+            strcpy(command.value, token);
+            break;
```

Comment:
When we use fixed length strings in the Command, the length of the token must be checked. Otherwise overflows are possible.
The [man page](https://linux.die.net/man/3/strcpy) suggests using `strncpy` or checking the token's length.